Makefile

```diff
@@ -4,6 +4,7 @@ DOCKER_TAG ?= latest
 PREFIX ?= /usr
 BINDIR ?= ${PREFIX}/bin
 UID=$(shell id -u)
+GOCACHEDIR=$(shell go env GOCACHE)
```

Comment:
Actually, this requires `go` binary on the host. However, I think it's a sane requirement.
Alternatively, we can do following:
```
GOCACHEDIR=$(shell which go >/dev/null 2>&1 && go env GOCACHE || echo "$(HOME)/.cache/go-build"
```

Reply:
Ok, done.